### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gasperTheGhost**
Updated API handler documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+**Every POST request checks for API key**
+Pass API key with `curl -X POST -d "apikey=\<apikey>" https://server.domain.com/<handler>`
+
+- POST /target
   Returns target to wich should be all the following requests sent
+  `curl -X POST -d "apikey=\<apikey>" https://server.domain.com/target`
   
-- GET /:target//counter 
+- POST /:target/counter 
   Returns current package number that should be calculated. First we empty the leftover list (filled every two hours with lookup to the ftp server), after all of those are taken, we continue on incrementing active counter.
```
Comment (pevecyan) [inline README.md]: We lookup for calculated files directly on server storage without FTP.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+  `curl -X POST -d "apikey=\<apikey>" https://server.domain.com/1/counter`
   
+- POST /:target/file/down/:counter 
+  Returns actual package with number passed as argument
+  `curl -X POST -d "apikey=\<apikey>" https://server.domain.com/1/file/down/16`
 
+- POST /:target/file/target/archive
+  Returns :target/targets/archive.zip which is downloaded from ftp only once and after that we use cached version.
```
Comment (pevecyan) [inline README.md]: not ftp, directly from disk

File: README.md (hunk 1 above)
Comment (gasperTheGhost) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (gasperTheGhost) [reply README.md]: Fixed

**Pull request merged**